src/Microsoft.Azure.WebJobs.Host/Executors/FunctionResult.cs

```diff
@@ -66,5 +66,10 @@ public FunctionResult(bool succeeded, Exception exception)
         /// Gets or sets the exception that occurred during function execution.
         /// </summary>
         public Exception Exception { get; private set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the function was exited early.
+        /// </summary>
+        public bool ReturnedEarly { get; set; }
```

Comment:
I don't think we should add this to this public type - the new property doesn't have much meaning in many of the existing usages of the type.

I think you're adding this to drive your logging stuff, so if you don't do the logging in this PR, this wouldn't be needed? Another way to do this would be for you to create your own subclass for use by the blob executor.